validate.py

```diff
@@ -136,6 +136,11 @@ def determine_if_winner():
 
   scalar = min(0.00001, 1.0 / total_points)
   points_so_far = 0
+  
+  print('Probability of winning:')
+  for user, user_points in summed_user_points:
+  print('%s: %.3f%%' % (user, user_points * scalar * 100))
```

Comment:
indentation is wrong

testing your PRs locally with `run.sh` would be a good idea!

Reply:
The problem is that if I open a PR through the GitHub UI, I can't push to it, so I'm copying & pasting any changes I make locally into the UI for subsequent changes. Maybe that just means I shouldn't open PRs through the UI, but there's other advantages to doing so.